node-serialport fails to compile on OpenBSD (with node-gyp rebuild)

```
../src/serialport_unix.cpp: In function 'int ToBaudConstant(int)':
../src/serialport_unix.cpp:50: error: 'B460800' was not declared in this scope
../src/serialport_unix.cpp:51: error: 'B500000' was not declared in this scope
../src/serialport_unix.cpp:52: error: 'B576000' was not declared in this scope
../src/serialport_unix.cpp:53: error: 'B921600' was not declared in this scope
../src/serialport_unix.cpp:54: error: 'B1000000' was not declared in this scope
../src/serialport_unix.cpp:55: error: 'B1152000' was not declared in this scope
../src/serialport_unix.cpp:56: error: 'B1500000' was not declared in this scope
../src/serialport_unix.cpp:57: error: 'B2000000' was not declared in this scope
../src/serialport_unix.cpp:58: error: 'B2500000' was not declared in this scope
../src/serialport_unix.cpp:59: error: 'B3000000' was not declared in this scope
../src/serialport_unix.cpp:60: error: 'B3500000' was not declared in this scope
../src/serialport_unix.cpp:61: error: 'B4000000' was not declared in this scope
gmake: *** [Release/obj.target/serialport/src/serialport_unix.o] Error 1
gmake: Leaving directory `/home/ian/ha/node_modules/rfxcom/node_modules/serialport/build'
```

These constants are not available on OpenBSD (as per Apple platforms). Commenting out these defines allows the source to compile.

--- a/src/serialport_unix.cpp
+++ b/src/serialport_unix.cpp
@@ -1,4 +1,3 @@
-
 #ifndef WIN32
 #include "serialport.h"
 #include <unistd.h>
@@ -46,7 +45,7 @@ int ToBaudConstant(int baudRate) {
     case 57600: return B57600;
     case 115200: return B115200;
     case 230400: return B230400;
-#ifndef __APPLE__
+#if !defined(__APPLE__) && !defined(__OpenBSD__)
     case 460800: return B460800;
     case 500000: return B500000;
     case 576000: return B576000;

